bears/c_languages/ClangBear.py

```diff
@@ -27,6 +27,7 @@ class ClangBear(LocalBear):
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Variable Misuse', 'Syntax'}
```

Comment:
I'd add `Semantics` (it's in the bear's description and it's a valid CAN_FIX value